compiler/tests-common/tests/org/jetbrains/kotlin/codegen/debugInformation/TestProcessServer.kt

```diff
@@ -0,0 +1,109 @@
+/*
```

Comment:
Why not merge this stuff with one in tests-common-jvm6?

Reply:
Merged into existing TestProcessServer with additional logic for handling output and port allocation.